### DISCUSSION TITLE
Automatic sample submission notification keeps bothering me

### DISCUSSION THREAD

**Discussion by Lutrus-P** [ChrisTitusTech/winutil#3135] [General]
After running standard tweaks of winutil which disables "Automatic Sample Submission" notification about it keeps refreshing every restart, even when dismissing from Defender app.

![image](https://github.com/user-attachments/assets/66ba779c-cd78-4af5-b1fd-91c19ca9cc84)

How to disable this notification permanently?

**Comment by michaelulmer** [comment]
Came here to post this. 
It is annoying because you always open defender due to the warning in the taskbar icon, just to see its only this minor thing...
Don't understand why Microsoft would reprompt this after you set it to "dismiss".. just MS bulltshit. 

however, I believe there are companies who disable this and also get the notification disabled. So it should be possible (hopefully) just through a simple registry value.
If I find something I will update this post

Update: I asked copilot online and got these results:

🧱 Option 2: Use Group Policy (for advanced users or IT environments)
If you're using Windows Pro or Enterprise, you can suppress the prompt via Group Policy:

1. Press Win + R, type gpedit.msc, and hit Enter
2. Navigate to: Computer Configuration > Administrative Templates > Windows Components > Microsoft Defender Antivirus > MAPS
3. Double-click Send file samples when further analysis is required
4. Set it to Enabled, then choose Never send or Send safe samples automatically

This disables automatic submission and suppresses the warning.

🧼 Option 3: Registry Edit (if Group Policy isn’t available)
Warning: Editing the registry can cause system issues if done incorrectly.

1. Press Win + R, type regedit, and hit Enter
2. Navigate to: HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender\Spynet
3. Create or modify the following DWORD values:
4. SubmitSamplesConsent = 2 (Never send)
5. SpynetReporting = 0 (Basic reporting)
6. Restart your PC to apply changes.

########################

I did Option 3 via registry and havent seen the warning after a reboot. Time will show if this message pops up again. Also I'm not sure where/how chris has this modification done. In the registry location (above) I didn't have the Key "Spynet" and also in the gpedit I didn't have any configuration in MAPS...

So its unclear if my settings are 1:1 as those from this the util and I didn't look up how chris implemented it in the first place
